recipes/vsnp/build.sh

```diff
@@ -2,4 +2,8 @@
 
 mkdir -p $PREFIX/bin
 mv bin/*.py $PREFIX/bin
-mv dependencies $PREFIX
+
+#prevent clobbering user's reference_options_paths.txt
+if [[ ! -f $PREFIX/dependencies/reference_options_paths.txt ]]; then
```

Comment:
This will always be installed, since a user's computer will never see this shell script. As a philosophy, dependencies should be installed by conda.

Reply:
okay, I'll change the build script and do another pull request.  Clobbering a users file won't be the worst thing.  They will just have to set it back up.